content/nginxaas-azure/getting-started/migrate-from-standard.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Migrating from Standard to Standard V2
-weight: 200
+title: Migrate from Standard to Standard V2
+weight: 1000
 toc: true
-url: /nginxaas/azure/troubleshooting/migrate-from-standard/
+url: /nginxaas/azure/getting-started/migrate-from-standard/
```

Comment:
This section does not seem appropriate for existing deployments.  I would suggest the `FAQ` section, but that seems to be gone too,

Reply:
Hi @russokj 

The objective of this PR is to remove the FAQ and Troubleshooting sections. The logic behind this change has been discussed with the team and @travisamartin extensively. 

We can follow up on where this document should live but the FAQ will not be an option going forward.